praw/__init__.py

```diff
@@ -102,6 +102,7 @@ class Config(object):  # pylint: disable=R0903
                  'flairselector':       'api/flairselector/',
                  'flairtemplate':       'api/flairtemplate/',
                  'friend':              'api/friend/',
+                 'friend_v1':           'api/v1/me/friends/%s',
```

Comment:
Can we use the newer _format_ syntax for new urls? At some point I want to change them all, but at least I'd like to avoid using the `%` style.